<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BJWindow</class>
 <widget class="QMainWindow" name="BJWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>qBlackJack</string>
  </property>
  <widget class="QWidget" name="root">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="GameSceneView" name="gameView">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::LosslessImageRendering|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::AnchorViewCenter</enum>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::AnchorViewCenter</enum>
      </property>
      <property name="viewportUpdateMode">
       <enum>QGraphicsView::FullViewportUpdate</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Card Skin</string>
    </property>
    <addaction name="loadCardSkinAction"/>
    <addaction name="resetCardSkinAction"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="resetGameAction"/>
   </widget>
   <widget class="QMenu" name="menuSound">
    <property name="title">
     <string>Sound</string>
    </property>
    <addaction name="toggleMusicAction"/>
    <addaction name="toggleSoundEffectsAction"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuSettings"/>
   <addaction name="menuSound"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="loadCardSkinAction">
   <property name="text">
    <string>Load from File</string>
   </property>
  </action>
  <action name="resetGameAction">
   <property name="text">
    <string>Reset Balance</string>
   </property>
  </action>
  <action name="resetCardSkinAction">
   <property name="text">
    <string>Reset to Default</string>
   </property>
  </action>
  <action name="toggleMusicAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Music</string>
   </property>
  </action>
  <action name="toggleSoundEffectsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sound Effects</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GameSceneView</class>
   <extends>QGraphicsView</extends>
   <header>widgets/gamesceneview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>loadCardSkinAction</sender>
   <signal>triggered()</signal>
   <receiver>BJWindow</receiver>
   <slot>doChangeCardSkin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetCardSkinAction</sender>
   <signal>triggered()</signal>
   <receiver>BJWindow</receiver>
   <slot>doResetSkin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetGameAction</sender>
   <signal>triggered()</signal>
   <receiver>BJWindow</receiver>
   <slot>doResetGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toggleMusicAction</sender>
   <signal>triggered(bool)</signal>
   <receiver>BJWindow</receiver>
   <slot>doSetMusicEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toggleSoundEffectsAction</sender>
   <signal>triggered(bool)</signal>
   <receiver>BJWindow</receiver>
   <slot>doSetSoundEffectsEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doChangeCardSkin()</slot>
  <slot>doResetSkin()</slot>
  <slot>doResetGame()</slot>
  <slot>doSetSoundEffectsEnabled(bool)</slot>
  <slot>doSetMusicEnabled(bool)</slot>
 </slots>
</ui>
